store/manager.go

```diff
@@ -83,45 +83,49 @@ func NewManager(
 }
 
 // Get ... fetches a value from a storage backend based on the (commitment mode, type)
-func (m *Manager) Get(ctx context.Context, key []byte, cm commitments.CommitmentMeta) ([]byte, error) {
-	switch cm.Mode {
-	case commitments.OptimismKeccak:
-
+func (m *Manager) Get(ctx context.Context,
+	versionedCert commitments.EigenDAVersionedCert,
+	cm commitments.CommitmentMode,
+) ([]byte, error) {
+	switch cm {
+	case commitments.OptimismKeccakCommitmentMode:
 		if m.s3 == nil {
 			return nil, errors.New("expected S3 backend for OP keccak256 commitment type, but none configured")
 		}
 
 		// 1 - read blob from S3 backend
 		m.log.Debug("Retrieving data from S3 backend")
-		value, err := m.s3.Get(ctx, key)
+		// TODO: using SerializedCert as the key to not change current behavior,
+		// but feels like we should be using the entire versionedCert (meaning including the version byte)?
```

Comment:
I agree, but it seems like it could be hard to make this change for data migration reasons

It doesn't seem cryptographically feasible that two SerializedCerts will match exactly, so I think it's ok.

Reply:
agree. should I just remove the TODO?

Reply:
[89a9305](https://github.com/Layr-Labs/eigenda-proxy/pull/379/commits/89a9305c3bdefcc039503146a0c4442468aaef28)